contrib/Docker/README.md

```diff
@@ -1,54 +1,62 @@
 # Docker
-Circle CI linux pch job uses the Dockerfile contained in the same folder as this README to create an image with the binaries built for linux and store that in the job artifacts.
-
-The instructions below expect basic knowledge about how to configure TrinityCore and how to use Docker:
-1. Click on the green ✔ next to each commit
-1. Scroll to "ci/circleci: pch" and click on "Details"
-1. Login to Circle CI if necessary. You might have to repeat again the previous steps after logging in to reach the correct page
-1. Click on "Artifacts" tab in Circle CI website
-1. Download the docker.tar.gz archive containing the docker image
-1. Load the image into your Docker host using
+
+The Circle CI Linux PCH job uses the Dockerfile contained in the same folder as this README to create an image with the binaries built for Linux, and stores that in the job artifacts.
```

Comment:
the job is actually called "pch" lowercase
![image](https://user-images.githubusercontent.com/1153754/104824343-2322a380-5851-11eb-977a-0a5908a8b7d0.png)

Reply:
Fair point, I will revert that part.